jbmc/src/jbmc/jbmc_parse_options.cpp

```diff
@@ -1029,6 +1029,13 @@ void jbmc_parse_optionst::help()
     "                              manifest (checked in this order), the behavior is\n" // NOLINT(*)
     "                              the same as running jbmc on the corresponding\n" // NOLINT(*)
     "                              class file."
+    " jbmc -jar jarfile            JAR file to be checked\n"
```

Comment:
:pick: As per guidelines, these help strings should be put in a macro to avoid copy pasting the same info :rabbit:

Reply:
Fully agreed. If it's ok to omit the program name from the help text then this seems be doable.